### DISCUSSION TITLE
Is it possible to target repositories with wildcard using self hosted option

### DISCUSSION THREAD

**Discussion by msmmichael** [renovatebot/renovate#20791] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

34.118.2

### If you're self-hosting Renovate, select which platform you are using.

Bitbucket Server

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Id like to target repos by their project. For example all repos in a project named MODULES. 

### What you tried so far.

I have tried using a wildcard but no success.

{
  "platform": "bitbucket-server",
  "endpoint": "https://bitbucket1.company.com",
  "username": "renovate-bot",
  "password": "process.env.RENOVATE_PASSWORD",
  "autodiscover": "false",
  "printConfig": true,
  "dryRun": false,
  "logLevel": "debug",
  "repositories": ['APP/config', 'MODULES/*'],
}

### Relevant debug logs

INFO: Repository started (repository=MODULES/*)
       "renovateVersion": "34.118.2"
ERROR: Repository is not found (repository=MODULES/*)
 INFO: Repository finished (repository=MODULES/*)
       "cloned": true,
       "durationMs": 68
 INFO: Renovate is exiting with a non-zero code due to the following logged errors
       "loggerErrors": [
         {
           "name": "renovate",
           "level": 50,
           "logContext": "584J-kn-9qp5P2qFYcGwB",
           "repository": MODULES/*",
           "msg": "Repository is not found"
         }
       ]

**Comment by viceice** [comment, marked as answer]
1. format your code
2. use autodiscover and corresponding filter[^1]

[^1]: https://docs.renovatebot.com/self-hosted-configuration/#autodiscoverfilter

**Reply by msmmichael** [reply]
Thanks i will try it out
